### PR TITLE
imported post file into app file and created POST route to create new…

### PR DESCRIPTION
**Pull request by angelo-james**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
 mongo = PyMongo(app)
 
 from routes.user import *
+from routes.post import *
```
Comment (developerX) [inline app.py:11]: this imports the routes from routes/post.py right?

File: routes/post.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+from app import app, mongo
+from flask_restful import Resource, Api
+from flask import jsonify, request
+from bson.objectid import ObjectId
+
+api = Api(app)
+
+class AddPost(Resource):
+  def post(self):
```
Comment (developerX) [inline routes/post.py:9]: this is what defines it as a post? if I replaced with get would it be a get request?

**Review by developerX — APPROVED** [review]
After you answer the questions please merge into master :) ship it 👍

File: app.py (hunk 1 above)
Comment (angelo-james) [reply app.py:11]: yes

File: routes/post.py (hunk 2 above)
Comment (angelo-james) [reply routes/post.py:9]: yes, if you change the post on line 9 to get in will change it to a get request.

**Pull request merged**